src/main/java/net/io/kino/service/ReservationService.java

```diff
@@ -1,16 +1,22 @@
 package net.io.kino.service;
 
+import com.braintreepayments.http.HttpResponse;
 import net.io.kino.model.Order;
 import net.io.kino.model.PersonalDetails;
 import net.io.kino.model.Ticket;
 import net.io.kino.model.TicketType;
 
+import java.time.LocalDate;
 import java.util.List;
 import java.util.Optional;
 
 public interface ReservationService {
     Order createOrder(List<Ticket> tickets, PersonalDetails client);
 
+    HttpResponse<com.paypal.orders.Order> createTransaction(Order order);
```

Comment:
Interfejs nie powinien zwracać detalu implementacyjnego

Reply:
Jak dla mnie to mogłoby istnieć tylko w ReservationServiceImpl jako prywatne i createOrder by to od razu wywoływał i otrzymane id wrzucał do Ordera i zwracał